src/storage/src/hummock/sstable/mod.rs

```diff
@@ -83,7 +101,7 @@ impl Sstable {
 
     #[inline]
     pub fn encoded_size(&self) -> usize {
-        8 /* id */ + self.meta.encoded_size()
+        8 /* id */ + self.meta.encoded_size() + self.blocks.iter().map(|block|block.data().len()).sum::<usize>()
```

Comment:
This means we will take cached block data size into consideration in meta cache. Is this expected?